tox.ini

```diff
@@ -0,0 +1,56 @@
+[tox]
+envlist = py36, py37, mypy
+
+# ------------- Dependancy setup ---------------------
+[deps]
+deps =
+    pytest
+    pytest-xdist
+    pytest-cov
+    pytest-travis-fold
+    tensorflow
+
+[mldeps-py36]
+deps =
+    {[deps]deps}
+    https://download.pytorch.org/whl/cpu/torch-1.3.1%2Bcpu-cp36-cp36m-win_amd64.whl ; sys_platform == 'win32'
+    torch == 1.3.1 ; sys_platform == 'darwin'
+    https://download.pytorch.org/whl/cpu/torch-1.3.1%2Bcpu-cp36-cp36m-linux_x86_64.whl ; sys_platform == 'linux'
+
+[mldeps-py37]
+deps =
+    {[deps]deps}
+    https://download.pytorch.org/whl/cpu/torch-1.3.1%2Bcpu-cp37-cp37m-win_amd64.whl ; sys_platform == 'win32'
+    torch == 1.3.1 ; sys_platform == 'darwin'
+    https://download.pytorch.org/whl/cpu/torch-1.3.1%2Bcpu-cp37-cp37m-linux_x86_64.whl ; sys_platform == 'linux'
```

Comment:
so this system really sucks in hangar. I know it's where you got this from, but might be worth looking into any other options that might exist? I'd love to use something a bit more elegant than explicitly including URLs in a tox ini script if you know of any options?

Reply:
AFIK, the elegant option to install pytorch is anaconda, everything else is done with messy URLs

Reply:
great... /s